content/docs/2.1/scalers/influxdb.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "InfluxDB"
+layout = "scaler"
+availability = "v2.1+"
+maintainer = "Community"
+description = "Scale applications based on InfluxDB queries"
++++
+
+### Trigger Specification
+
+This specification describes the `InfluxDB` trigger that scales based on the results of a flux query.
+
+```yaml
+triggers:
+  - type: influxdb
+    metadata:
+      serverURL: http://influxdb:8086
+      organizationName: influx-org
+      organizationNameFromEnv: INFLUXDB_ORG_NAME
```

Comment:
Can we comment this one out, or at least mention that you either have to choose between `organizationName` or `organizationNameFromEnv` and not both please?

Reply:
@tomkerkhove Yeah no problem. I was following the Redis List example here: https://keda.sh/docs/2.0/scalers/redis-lists/

Which is why I added both, but I will comment the `env` ones out.

Reply:
Thanks! Will update those asap then to align as well.